cg/services/orders/storing/implementations/metagenome_order_service.py

```diff
@@ -55,32 +55,35 @@ def store_order_data_in_status_db(
         db_order: DbOrder = self.status_db.add_order(
             customer=customer, ticket_id=order._generated_ticket_id
         )
-        priority: PriorityEnum = order.samples[0].priority
-        db_case = self._create_db_case(order=order, customer=customer, priority=priority)
-        db_order.cases.append(db_case)
         with self.status_db.session.no_autoflush:
             for sample in order.samples:
+                db_case = self._create_db_case_for_sample(
+                    order=order, sample=sample, customer=customer
+                )
                 db_sample = self._create_db_sample(order=order, sample=sample, customer=customer)
-                new_relationship: CaseSample = self.status_db.relate_sample(
+                case_sample: CaseSample = self.status_db.relate_sample(
                     case=db_case, sample=db_sample, status=StatusEnum.unknown
                 )
-                self.status_db.add_item_to_store(new_relationship)
+                self.status_db.add_item_to_store(case_sample)
                 new_samples.append(db_sample)
-        self.status_db.add_item_to_store(db_case)
+                db_order.cases.append(db_case)
         self.status_db.add_item_to_store(db_order)
         self.status_db.add_multiple_items_to_store(new_samples)
         self.status_db.commit_to_store()
         return new_samples
 
-    def _create_db_case(
-        self, order: OrderMetagenome, customer: Customer, priority: PriorityEnum
+    def _create_db_case_for_sample(
+        self, order: OrderMetagenome, sample: SampleMetagenome, customer: Customer
     ) -> DbCase:
+        """Return a Case database object for a sample."""
+        ticket_id: str = str(order._generated_ticket_id)
+        case_name: str = f"{sample.name}-{ticket_id}"
         db_case: DbCase = self.status_db.add_case(
             data_analysis=ORDER_TYPE_WORKFLOW_MAP[order.order_type],
```

Comment:
```suggestion
            data_analysis=Workflow.RAW_DATA,
```
Something to consider